fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/test/spec/R4JDBCExamplesTest.java

```diff
@@ -44,10 +48,29 @@ public void bootstrapAndLoad() {
 
     @Test(groups = { "jdbc-seed" })
     public void perform() throws Exception {
-
-        R4JDBCExamplesProcessor processor = new R4JDBCExamplesProcessor(persistence,
-            () -> createPersistenceContext(),
-            () -> createHistoryPersistenceContext());
+        // Use connection pool and transaction provider to make sure the resource operations
+        // of each resource are committed after the processing is finished, and because this
+        // testng test process the samples one by one, so set the connection pool size to 1.
+        PoolConnectionProvider connectionPool = new PoolConnectionProvider(database, 1);
+        ITransactionProvider transactionProvider = new SimpleTransactionProvider(connectionPool);
+        List<ITestResourceOperation> operations = new ArrayList<>();
+        operations.add(new CreateOperation());
+        operations.add(new ReadOperation());
+        operations.add(new UpdateOperation());
+        operations.add(new UpdateOperation());
+        operations.add(new ReadOperation());
+        operations.add(new VReadOperation());
+        operations.add(new HistoryOperation(3));
+        operations.add(new DeleteOperation());
+        operations.add(new DeleteOperation());
+        operations.add(new HistoryOperation(4));
+        R4JDBCExamplesProcessor processor = new R4JDBCExamplesProcessor(
+                operations,
+                this.properties,
+                connectionPool,
+                null,
+                null,
```

Comment:
is it necessary to have null here? on 71/27?

Reply:
maybe cleanup the class?

Reply:
this is a shared function used by both db2 and derby, used to pass tenant names and key for db2 test.